package.yaml

```diff
@@ -1,10 +1,10 @@
 name: closed
-version: "0.2.0"
+version: "0.2.1"
```

Comment:
Should this be `0.2.0.1`?

Reply:
This package has used SemVer instead of PVP since the beginning. Do you think it's save to retroactively make this PVP?

Reply:
:scream: I have no idea. @pbrisbin what are your thoughts? If this is a jump to PVP from SEMVER then I'd expect an epoch bump.

```
EPOCH.MAJOR.MINOR.PATCH
```

1. A rewrite or major shift in API happens in EPOCH
2. Breaking changes may only happen in MAJOR (or EPOCH)
3. New (non-breaking) API additions happen in MINOR
4. Bug fixes that don't effect the API happen in PATCH

Reply:
Or we could just consider `0.2` to be major version `0.2` within PVP.

Reply:
I'm not sure if there's any report-specified answer, but empirically "up-converting" a 3-part version to a 4-part version just has to use trailing zeros to ensure the bigger version is considered newer than the smaller:

```
λ> :m +Data.Version
λ> makeVersion [0, 2, 0] `compare` makeVersion [0, 2, 0, 0]
LT
λ> makeVersion [0, 2, 0] `compare` makeVersion [0, 2, 0, 1]
LT

Using leading zeros gets it wrong:

```
λ> makeVersion [0, 2, 0] `compare` makeVersion [0, 0, 2, 0]
GT
λ> makeVersion [0, 2, 0] `compare` makeVersion [0, 0, 2, 1]
GT
```

So if we want to release a 4-part version (and I'd like to, so we align with present-day guides on the matter), we should pretend the current version is `0.2.0.0` and then go from there.

I think releasing as `0.2.0.1` would be safe and accurate w.r.t. "breaking-ness". But if you all want to bump MAJOR or EPOCH anyway, that doesn't bother me at all.

Reply:
`0.2.0.1` works for me, I'll update it.

Reply:
Yeah, that works for me :+1: